iree/tools/iree-benchmark-module-main.cc

```diff
@@ -350,10 +422,6 @@ int main(int argc, char** argv) {
                                IREE_FLAGS_PARSE_MODE_CONTINUE_AFTER_HELP,
                            &argc, &argv);
   ::benchmark::Initialize(&argc, argv);
-  std::cout << "NOTE: IREE is not a BLAS library and this tool is intended\n"
-            << "to be used when benchmarking entire programs. Precise timing\n"
-            << "of automatically generated device-side execution is not\n"
-            << "available here.\n";
```

Comment:
Some message may still be useful to log, maybe in each of the branches of

```c++
      if (iree_string_view_equal(benchmark_type, IREE_SV("dispatch"))) {
         ...
      } else if (iree_string_view_equal(benchmark_type, IREE_SV("entry"))) {
         ...
```

WDYT?

Reply:
Oh, and/or update https://github.com/google/iree/blob/main/docs/developers/developing_iree/benchmarking.md? (the PR description is a good start, could even link to this PR in the docs)